src/node_buffer.cc

```diff
@@ -482,17 +485,20 @@ void StringSlice(const FunctionCallbackInfo<Value>& args) {
 
 
 template <>
-void StringSlice<UCS2>(const FunctionCallbackInfo<Value>& args) {
+inline void StringSlice<UCS2>(const FunctionCallbackInfo<Value>& args,
+                              Local<Value> buffer_arg,
+                              Local<Value> start_arg,
+                              Local<Value> end_arg) {
   Isolate* isolate = args.GetIsolate();
```

Comment:
Perhaps we could match `StringSlice()` above and instead do `Isolate* isolate = env->isolate();` for consistency?